modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuResourceImpl.java

```diff
@@ -343,9 +343,17 @@ private Sku _updateSKU(CPInstance cpInstance, Sku sku) throws Exception {
 
 		DateConfig expirationDateConfig = new DateConfig(expirationCalendar);
 
+		String externalReferenceCode = sku.getExternalReferenceCode();
+
+		if ((externalReferenceCode == null) ||
+			externalReferenceCode.isEmpty()) {
+
+			externalReferenceCode = cpInstance.getExternalReferenceCode();
+		}
+
```

Comment:
Hi @victorcdp , good try for a first attempt. I wanted you to go through this just to implement the condition as practice.
Please update PR accordingly by using `GetterUtil.get` instead. Example usage can be seen directly in the update call.

Reply:
To further clarify, the change you have made is redundant because `GetterUtil.get` is a util helper class that handles such checks for you.